airflow/timetables/interval.py

```diff
@@ -76,21 +80,23 @@ def next_dagrun_info(
         if not restriction.catchup:
             earliest = self._skip_to_latest(earliest)
         elif earliest is not None:
-            earliest = self._align(earliest)
+            earliest = self._align_to_next(earliest)
         if last_automated_data_interval is None:
             # First run; schedule the run at the first available time matching
             # the schedule, and retrospectively create a data interval for it.
             if earliest is None:
                 return None
             start = earliest
-        else:  # There's a previous run.
+        else:  # There's a previous run
```

Comment:
Accidental?

Reply:
sorry !!!   Thank you for your review, I made changes. Looking forward to your new comment.

Reply:
Hello